deploy/charts/openebs-monitoring/dashboards/Control plane/openebs-main-dashboard.json

```diff
@@ -0,0 +1,1726 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "This dashboard gives overview of OpenEBS installation in cluster.",
```

Comment:
n_: `This dashboard gives an overview of OpenEBS cStor status in the cluster.`

Reply:
Fixed